Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -16,44 +16,41 @@
 
 #include "Starboard.h"
 #import <StubReturn.h>
+#include <vector>
 
-#include "Foundation/NSOrderedSet.h"
+#import <Foundation/NSOrderedSet.h>
+#import <Foundation/NSException.h>
+
+@implementation NSOrderedSet {
+    std::vector<StrongId<id>> setContainer;
```

Comment:
I came here to say that; the ideal case for an ordered set might be:

set<StrongId> set
unordered_map<id, set::iterator>

with a std::hash specialization for id that uses -[NSObject hash]

---
In reply to: [61937213](https://github.com/Microsoft/WinObjC/pull/472#discussion_r61937213) [](ancestors = 61937213)